cli/cmd/generate_gcp.go

```diff
@@ -0,0 +1,787 @@
+package cmd
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+
+	"github.com/lacework/go-sdk/internal/file"
+	"github.com/lacework/go-sdk/lwgenerate/gcp"
+)
+
+var (
+	// Define question text here to be reused in testing
+	QuestionGcpEnableConfig            = "Enable Config Integration?"
+	QuestionGcpEnableAuditLog          = "Enable AuditLog Integration?"
```

Comment:
Based on Google's doc, it is Audit Logs - https://cloud.google.com/logging/docs/audit

```suggestion
	QuestionGcpEnableAuditLog          = "Enable Audit Logs integration?"
```

Reply:
This is another place where I have named this to be in line with our TF module name.

Reply:
I'm going to leave this as is to keep us consistent with our Docs and module names.

Reply:
I agree with the Audit Log being two words.